src/test/kotlin/org/rust/lang/core/completion/RsPathCompletionFromIndexTest.kt

```diff
@@ -6,14 +6,16 @@
 package org.rust.lang.core.completion
 
 import org.intellij.lang.annotations.Language
+import org.rust.ProjectDescriptor
+import org.rust.WithDependencyRustProjectDescriptor
 import org.rust.hasCaretMarker
 import org.rust.ide.settings.RsCodeInsightSettings
 import org.rust.lang.core.completion.RsCommonCompletionProvider.Testmarks
 import org.rust.openapiext.Testmark
 
 class RsPathCompletionFromIndexTest : RsCompletionTestBase() {
 
-    fun `test suggest an non-imported symbol from index and add proper import`() = doTest("""
+    fun `test suggest an non-imported symbol from index and add proper import`() = doTestByText("""
```

Comment:
We usually use names like `...byCode`, isn't it?

Reply:
In `RsAnnotationTestBase` we use `byText` and `byFileTree` suffix, i.e. all annotator and inspection tests uses base methods with such prefixes